src/core/ui.js

```diff
@@ -197,20 +196,18 @@ export const ui = {
   enable() {
     respecPill.removeAttribute("disabled");
   },
-  addCommand(label, handler, keyShort, icon) {
+  /**
+   * @param {string} _keyShort shortcut key. unused - kept for backward compatibility.
+   */
+  addCommand(label, handler, _keyShort, icon) {
```

Comment:
Some users of `window.respecUI.addCommand`:
- https://github.com/w3c/core-aam/blob/3e4d427f227be9bdb05f929c5eb74e0ea4a0f5e6/common/script/aria.js#L17
- https://github.com/w3c/graphics-aria/blob/aae7c6f397d9ea66911e07c084faff27bba47c52/common/script/aria.js#L17
- https://github.com/w3c/math-aam/blob/bf5d8225dbf0446e9feb4279e443baed187381bf/common/script/aria.js#L17